aip/general/0193.md

```diff
@@ -62,11 +62,109 @@ a [`PreconditionFailure`][PreconditionFailure].
 **Note:** `ErrorInfo` represents a special case. There **must** be exactly one
 `ErrorInfo`. It is required.
 
-### Localization
-
-Error messages **must** be in English. If a localized error message is also
-required, the service **should** use [`google.rpc.LocalizedMessage`][details]
-in the `details` field.
+### Error messages
+
+For each error, the service **must** populate the `message` field on [`google.rpc.Status`][Status]. This error message,
+
+- is developer-facing, human-readable, and in the application's
+  native language
+- both explains the error and offer an actionable resolution to it
+  ([citation](https://cloud.google.com/apis/design/errors#error_model))
+
+**Note:** Sometimes, services elect to publish debug error message in
+English rather than the application's native language so that messages
+are more easily universally searchable in common online knowledge bases.
+
+For new errors, the payload **must** include `ErrorInfo` and any
+variables found in dynamic segments of the error message **must** be
+represented in `metadata`. See, "[Dynamic
+variables](#dynamic-variables)".
+
+#### Changing error messages
+
+Changes to `message` on `Status` **must not** change it if your
+service's API design governing body decides that to do so represents a
+breaking change and **should not** change it the text contains dynamic
```

Comment:
I'm sorry I didn't grok this paragraph. Can this be reworded to be a bit more concise and separated? Something like,

```
If your service's API design governing body decides that changing a `Status.message` is a
breaking change, then the `Status.message` **must not** be changed.

If the `Status.message` contains dynamic data, but `ErrorInfo` was absent at the time of
launch, then the `Status.message` **should not** be changed.

To present a different, potentially more detailed error, consider using the
following approach.

// Go on to explain how that works...
```

Note: I used `Status.message`, but you can use your "message on Status" style too idc

Reply:
Done.